dkg/mod.go

```diff
@@ -17,7 +17,7 @@ type Actor interface {
 	// Setup must be first called by ONE of the actor to use the subsequent
 	// functions. It creates the public distributed key and the private share on
 	// each node. Each node represented by a player must first execute Listen().
-	Setup(players mino.Players, pubKeys []kyber.Point, threshold int) error
+	Setup(players mino.Players, pubKeys []kyber.Point, threshold int) (pubKey kyber.Point, err error)
```

Comment:
Can you implement a `crypto.CollectiveAuthority` that wraps the addresses and the public keys ? You can implement the crypto ed25519 abstraction in the same time.